src/main/scala/org/holmesprocessing/totem/driver/driver.scala

```diff
@@ -138,6 +140,8 @@ object driver extends App with Instrumented {
           PEiDWork(key, uuid_filename, taskingConfig.default_service_timeout, "PEID", GeneratePartial("PEID"), li)
         case ("PEINFO", li: List[String]) =>
           PEInfoWork(key, uuid_filename, taskingConfig.default_service_timeout, "PEINFO", GeneratePartial("PEINFO"), li)
+        case ("SHODAN", li: List[String]) =>
+          ShodanWork(key, orig_filename, List(taskingConfig.default_service_timeout, 300).max, "SHODAN", GeneratePartial("SHODAN"), li)
```

Comment:
and for clarity, if you have a solid reason I am all ear. just dont know if the added complexity is worth it